src/Raven.Client/Documents/Changes/DatabaseChanges.cs

```diff
@@ -651,10 +678,10 @@ private async Task ProcessChanges()
                             {
                                 try
                                 {
-                                    if (json.TryGet(nameof(TopologyChange), out bool supports) && supports)
+                                    if (json.TryGet(nameof(TopologyChange), out bool _))
                                     {
-                                        GetOrAddConnectionState("Topology", "watch-topology-change", "", "");
```

Comment:
trying to understand if we are not changing the behavior here... each time we received a TopologyChange, we were subscribing to topology changes and updating topology? Now we will do it once only?

Reply:
There is no behavior change here.
We are actually only sending this _once_, see `SendSupportedFeatures` at the start of the connection.
We added it in this way for backward compact scenario, I believe. This just make it explicit and make it easier to add more features / behaviors down the line.